### ISSUE TITLE
Backpropagation of the loss function

### ISSUE THREAD

**Issue by Loshhhhh** [OrigamiSL/FDNet2023#1]
Hi，
May I ask why torch.ones_like(loss) is used when doing backpropagation of the loss function ？
Thank you！

**Comment by OrigamiSL** [comment]
That makes no difference to directly use 'loss' for propagation in this project where loss is a number. However, the function torch.ones_like() is needed if loss is a tensor with size bigger than one. I am accustomed to such usage in that the preceding occasion occurs in some other tasks.
